app/web/package.json

```diff
@@ -93,13 +97,13 @@
     "eslint-config-next": "12.2.5",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-simple-import-sort": "^12.1.1",
-    "eslint-plugin-storybook": "^0.11.0",
+    "eslint-plugin-storybook": "^0.10.0",
     "eslint-plugin-unused-imports": "^4.1.4",
     "i18next-resources-to-backend": "^1.0.0",
     "jest": "^28.1.3",
     "jest-environment-jsdom": "^29.7.0",
     "jest-junit": "^14.0.1",
-    "msw": "^0.42.0",
+    "msw": "^1.2.1",
```

Comment:
Needed to update these as they were causing incompatible Typescript types after the mui upgrade with the versions being so mismatched.